src/cosmos/gov/v1beta1/tx.ts

```diff
@@ -523,32 +598,34 @@ export const MsgDepositResponse = {
     return message;
   },
 };
+/** Msg defines the RPC service */
 
-/** Msg defines the bank Msg service. */
 export interface Msg {
-  /** SubmitProposal defines a method to create new proposal given a content. */
   SubmitProposal(request: MsgSubmitProposal): Promise<MsgSubmitProposalResponse>;
-  /** Vote defines a method to add a vote on a specific proposal. */
```

Comment:
I think it could be worth taking a 2nd look at comments — but I recall having issues controlling leading/trailing comments and it being an internal issue with `babel/generate`